Troubleshooting_GUI/install.sh

```diff
@@ -39,4 +39,4 @@ sudo chmod +x /home/pi/Desktop/Troubleshooting_Start.desktop
 
 delete_folder /home/pi/GoBox/Troubleshooting
 
-feedback "Done with TRoubleshooting"
\ No newline at end of file
+feedback "Done with TRoubleshooting"
```

Comment:
Since the file is already checked out, could you fix this typo here? (Capital R should be lowercase)

Reply:
Done.